packages/loot-core/src/client/actions/modals.ts

```diff
@@ -52,3 +55,28 @@ export function closeModal(): CloseModalAction {
 export function collapseModals(rootModalName: string) {
   return { type: constants.COLLAPSE_MODALS, rootModalName };
 }
+
+export function openAccountCloseModal(accountId: AccountEntity['id']) {
```

Comment:
This was originally in `desktop-client/src/actions/queries` but moved it here since it's modal related. No changes done on the function logic.